README.md

```diff
@@ -223,10 +223,11 @@ import Share from 'react-native-share';
 const url = 'https://awesome.contents.com/';
 const title = 'Awesome Contents';
 const message = 'Please check this out.';
+const icon = 'data:<data_type>/<file_extension>;base64,<base64_data>';
```

Comment:
Only works with base64?

Reply:
I tested with url. It overrides link metadata when image from url loaded.

Reply:
Maybe we can add something like:

```suggestion
const icon = 'data:<data_type>/<file_extension>;base64,<base64_data>' or "remote-image-url";
```

This should work?

Reply:
When I use remote url it loads title and url from internet and overrides linkMetadata.
```jsx
const icon = 'https://www.sercanuste.com/img/logo/BA.png';
```
![IMG_162F379A8183-1](https://user-images.githubusercontent.com/5119317/78928125-be0f7580-7aa8-11ea-9e09-d375106200ae.jpeg)